docs/kubernetes/how-to-deploy-kubeflow-on-linode-kubernetes-engine/index.md

```diff
@@ -0,0 +1,252 @@
+
+[Kubeflow](https://www.kubeflow.org/) is an excellent alternative for training and evaluating machine learning models in public and private clouds.
+
+Kubeflow is designed to make your machine learning experiments portable and scalable.
+
+You start by creating [Jupyter notebooks](https://jupyter.org/) in the cloud.
+
+When you sense that you have hit a breakthrough, you can scale your model to run on thousands of machines.
+
+Kubeflow optimises your model and breaks it down into smaller tasks that can be processed in parallel.
+
+Then, it distributes the tasks to several computers and waits until the results are ready.
+
+_Are you ready to train your model at scale?_
+
+## Before You Begin
+
+1.  Please notice that you can run this tutorial locally using [minikf](https://www.kubeflow.org/docs/started/workstation/getting-started-minikf/). However, you should have at least 12GB of RAM, 2CPUs and 50GB of disk space available.
+
+2.  The `kfctl` CLI used to deploy Kubeflow currently only works on Linux and Mac. If you're on Windows, you can use WSL2 or a Docker container to work around this limitation.
+
+3.  Most Kubeflow pipelines require [Kubernetes Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes) that can be attached to several nodes at once (ReadWriteMany). Currently, the only mode supported by the Linode Block Storage CSI driver is ReadWriteOnce, meaning that it can only be connected to one Kubernetes node at a time.
+
+## Create an LKE Cluster
+
+Follow the instructions in [Deploying and Managing a Cluster with Linode Kubernetes Engine Tutorial](https://www.linode.com/docs/kubernetes/deploy-and-manage-a-cluster-with-linode-kubernetes-engine-a-tutorial/) to create and connect to an LKE cluster.
+
+The [official Kubeflow documentation](https://www.kubeflow.org/docs/started/k8s/overview/) recommends provisioning a cluster with at least 4 CPU, 12GB of memory and 50GB of space available.
+
+We recommend running three "Linode 16GB" nodes — that should give you enough resources to scale your models.
+
+![Creating a Kubernetes cluster on Linode](create-cluster.png)
+
+You can verify that the installation is successful with:
+
+```bash
+kubectl get nodes
+```
+
+The output should similar to:
+
+{{< output >}}
+NAME                        STATUS   ROLES    AGE   VERSION
+lke7189-9006-5f05145fc9a3   Ready    <none>   8h    v1.17.3
+lke7189-9006-5f051460a1e2   Ready    <none>   8h    v1.17.3
+lke7189-9006-5f0514617a87   Ready    <none>   8h    v1.17.3
+{{</ output>}}
+
+## Install Kubeflow
+
+To install Kubeflow, you need three parts:
+
+1. A Kubernetes cluster, which you already provisioned in the previous step.
+1. `kfctl` — a command-line tool that configures and deploy Kubeflow.
+1. A `KfDef` file — a recipe of components that should be included in your Kubeflow installation.
+
+You can [download and install `kfctl` from the official repository](https://github.com/kubeflow/kfctl/releases/).
```

Comment:
1. Download the latest release of kfctl v1.0.2 from the Kubeflow [releases page] (https://github.com/kubeflow/kfctl/releases/).

2. Unpack the tar ball

tar -xvf kfctl_v1.0.2_<platform>.tar.gz

3. Add the location of kfctl binary to the path environment variable. If you don’t add the location of the binary to the path variable, you must use the full path to the kfctl binary each time you run it.

export PATH=$PATH:<path to where kfctl was unpacked>

Reply:
Amended